src/components/draft/editor/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from '@mui/material';
+import EditorFileSelector from 'components/draft/editor/FileSelector';
```

Comment:
Why not go with `FileSelector`?

Reply:
The `EditorFileSelector` ? It does match as it follows the pattern of including the parent folder when the component name is pretty non-descriptive.